internal/vm/remotevm/builder.go

```diff
@@ -40,7 +40,8 @@ func NewUVMBuilder(ctx context.Context, id, owner, binPath, addr string, guestOS
 		opts := &jobobject.Options{
 			Name: id,
 		}
-		job, err := jobobject.Create(ctx, opts)
+		var err error
```

Comment:
nit: Maybe we can declare this next to `jobobject` after line 33 and then the entire function can use the same variable?